go/test_multipart.sh

```
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# テスト用ヘルパースクリプト
+
+# カラー定義
+GREEN='\033[0;32m'
+RED='\033[0;31m'
```

Shellcheck (0.10.0)

[warning] RED appears unused. Verify use (or export if used externally).

(SC2034)